app/src/main/java/clarkson/ee408/tictactoev4/PairingActivity.java

```diff
@@ -68,58 +86,166 @@ protected void onCreate(Bundle savedInstanceState) {
      * Send UPDATE_PAIRING request to the server
      */
     private void getPairingUpdate() {
-        // TODO:  Send an UPDATE_PAIRING request to the server. If SUCCESS call handlePairingUpdate(). Else, Toast the error
+        // Create Request object with type UPDATE_PAIRING
+        Request request = new Request();
+        request.setType(RequestType.UPDATE_PAIRING);
+
+        // TODO: Send an UPDATE_PAIRING request to the server. If SUCCESS call handlePairingUpdate(). Else, Toast the error
```

Comment:
Remove "TODO" marker so it doesn't get flagged by Android Studio

```suggestion
        // Send an UPDATE_PAIRING request to the server. If SUCCESS call handlePairingUpdate(). Else, Toast the error
```